rules_haskell_tests/WORKSPACE

```diff
@@ -48,13 +48,12 @@ rules_proto_dependencies()
 rules_proto_toolchains()
 
 # For buildifier
-# starting from 0.29, rules_go requires bazel >= 4.2.0
 http_archive(
```

Comment:
@ylecornec do you know which Bazel version rules_go 0.39.1 requires? AFAIK, 0.35 requires at least Bazel 5.1, so we might want to note somewhere that we require at least version x.y for developing on this repo? Or just use a `.bazelversion` file?

Reply:
Ah indeed, it requires at least Bazel `5.3`. 

Release [0.38](https://github.com/bazelbuild/rules_go/releases/tag/v0.38.0) is the one with the fix for `incompatible_disable_starlark_host_transitions` and requiring this version.